<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>239</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>239</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Check User Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {margin:0; padding:0; border: 0}

QDialog{background-color:#efefef}

QPushButton{
	background-color: #ffdf00;
	border: 0;
border-radius:25; padding:0; margin:0
}
QPushButton::disabled{background-color:#cbcbcb}
QPushButton:hover{ background-color: #f1d100}
QPushButton:pressed{background-color:#dfc200 }

QLineEdit{
background-color: white;
border: 2px solid #b0b0b0;
border-radius: 18
}
QLineEdit {\ncolor: black; \nbackground-color: white;\nborder-width: 1px;\nborder-radius: 25px;\n}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>208</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Selecione a User Frame:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uf_num">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>208</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Selecione um tamanho da fresa:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="offset_num">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_confirm">
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>175</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Confirmar</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/assets/icons/sharp_check_black_48dp.png</normaloff>:/icons/assets/icons/sharp_check_black_48dp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>175</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">* {
background-color: #eece00
}

QPushButton:hover{
background-color: #ddbd00
}

QPushButton:pressed {
background-color: #ccac00
}
QPushButton::disabled{background-color:#cbcbcb}</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/assets/icons/sharp_close_black_48dp.png</normaloff>:/icons/assets/icons/sharp_close_black_48dp.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
